frontend/pages/review.tsx

```diff
@@ -106,136 +152,129 @@ function ApprovalList({ category, filter }: { category: ApprovalCategory; filter
-                      </Alert>
-                    ))}
-                </>
-              )}
-              {category === 'Deployment' && (
-                <>
-                  <Link href={`/deployment/${approvalObj.deployment?.uuid}`} passHref legacyBehavior>
-                    <MuiLink
-                      variant='h5'
-                      sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}
-                    >
-                      {approvalObj.deployment?.metadata?.highLevelDetails?.name}
-                    </MuiLink>
-                  </Link>
-                  <Typography variant='body1'>
-                    Requesting deployment of{' '}
-                    <Link href={`/model/${approvalObj.deployment?.model?.uuid}`} passHref legacyBehavior>
-                      <MuiLink sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}>
-                        {approvalObj.deployment?.model?.latestVersion?.metadata?.highLevelDetails?.name}
+    <>
+      {approvalObj && (
+        <>
+          <Box sx={{ px: 3 }} key={approvalObj._id}>
+            <Grid container spacing={1} sx={approvalObj.approvalType === 'Manager' ? managerStyling : reviewerStyling}>
+              <Grid item xs={9}>
+                {category === 'Upload' && (
+                  <>
+                    <Link href={`/model/${approvalObj.version?.model?.uuid}`} passHref legacyBehavior>
+                      <MuiLink
+                        variant='h5'
+                        sx={{ fontWeight: '500', textDecoration: 'none', color: theme.palette.secondary.main }}
+                      >
+                        {approvalObj.version?.metadata?.highLevelDetails?.name}
                       </MuiLink>
                     </Link>
-                  </Typography>
-                  {approvalObj.deployment === undefined ||
-                    (approvalObj.deployment === null && (
-                      <Alert sx={{ mt: 2 }} severity='warning'>
-                        This deployment appears to have data missing - check with the requester to make sure it was
-                        requested correctly
-                      </Alert>
-                    ))}
-                </>
-              )}
-            </Grid>
-            <Grid item xs={12} md sx={{ display: 'flex' }}>
-              <Box ml='auto' my='auto'>
-                {approvalObj.status !== 'No Response' && (
-                  <Chip label={approvalObj.status} color={approvalObj.status === 'Accepted' ? 'success' : 'error'} />
+                    <Stack direction='row' spacing={2}>
+                      <Chip color='primary' label={approvalObj.approvalType} size='small' />
+                      <Chip color='primary' label={`Version: ${approvalObj.version?.version}`} size='small' />
+                      <Box sx={{ mt: 'auto !important', mb: 'auto !important' }}>
+                        <Typography variant='body1'>
```

Comment:
The default value of `variant` is `body1`, so this can just be
```
<Typography>
    {approvalObj.version?.metadata?.highLevelDetails?.modelInASentence}
</Typography>
```